src/meshapi/views/forms.py

```diff
@@ -101,6 +109,21 @@ def join_form(request: Request) -> Response:
         logging.exception("TypeError while processing JoinForm")
         return Response({"detail": "Got incomplete form request"}, status=status.HTTP_400_BAD_REQUEST)
 
+    if not settings.DEBUG and not DISABLE_RECAPTCHA_VALIDATION:
+        try:
+            request_source_ip, request_source_ip_is_routable = get_client_ip(request)
+            if not request_source_ip_is_routable:
+                request_source_ip = None
```

Comment:
Add logging statement here to check that our reverse proxy stuff is working right

Reply:
See https://github.com/nycmeshnet/meshdb/pull/712